extensions/wikipedia/src/wikipedia.ts

```diff
@@ -14,6 +15,20 @@ export interface PageSummary {
   };
 }
 
+interface Preferences {
+  zhVariant: string;
+}
+
+const preferences = getPreferenceValues<Preferences>();
+
+function getVariant(language: string) {
+  if (language === "zh") {
+    return preferences.zhVariant;
+  }
+
+  return "";
+}
```

Comment:
As we won't always have a variant, I suggest explicitly returning `undefined` in this case.

```suggestion
function getVariant(language: string) {
  if (language === "zh") {
    return preferences.zhVariant;
  }

  return undefined;
}
```